ManatiCR/generator-mdsk
# 57: Get latest drupal 8 version automatically.

Fixes #57 
Default to 8.3.2; but get the latest version automatically.

How to test:
- Generate new drupal 8 project
- Stop composer and npm install. You don't need to provision the vm.
- Verify composer.drupal.json: drupal/core should have the latest drupal version available.
